### DISCUSSION TITLE
Workflow Place Condition or Transition Guard for Admin users

### DISCUSSION THREAD

**Discussion by EliShklovsky** [pimcore/pimcore#17602] [Q&A / Support] [answered]
Hi
Is there a way to query Admin users on Workflow Place Condition or Transition guard:
Something like:

`(is_fully_authenticated() and 'ROLE_WF_EDITORS' in role_names) or (is_fully_authenticated() and  'ROLE_ADMIN' in role_names)`

Using ROLE_ADMIN does not work.
Thanks

**Comment by nevo-netgo** [comment, marked as answer]
Hi Eli,
try using 'ROLE_PIMCORE_ADMIN' instead of 'ROLE_ADMIN'.
